app/views/centers/show.html.slim

```diff
@@ -0,0 +1,20 @@
+h2
+  | Details of #{@center.name}
```

Comment:
change this to `h2 Details of #{@center.name}` to make it consistent w/ the other pages